src/operator-docs/combination/forkJoin.ts

```diff
@@ -1,6 +1,139 @@
 import { OperatorDoc } from '../operator.model';
 
 export const forkJoin: OperatorDoc = {
-  'name': 'forkJoin',
-  'operatorType': 'combination'
+  name: 'forkJoin',
+  operatorType: 'combination',
+  signature: 'public static forkJoin(sources: *): any',
+  parameters: [
+    {
+      name: 'sources',
+      type: '*',
```

Comment:
The type here should be an observable ?

Reply:
Technically no - in RxJS we have a type called Observable input. It is really Observable or observable or array or iterable or promise. 

So we might want to have a way in the docs to represent this. In the docs it might be good to have a location to where it explains that all of these things can be returned anywhere when you accept an Observable that we also accept these.